src/encoded_core/tests/conftest.py

```diff
@@ -0,0 +1,427 @@
+"""py.test fixtures for Pyramid.
+
+http://pyramid.readthedocs.org/en/latest/narr/testing.html
+"""
+
+# import datetime as datetime_module
+import logging
+import os
+import pkg_resources
+import pytest
+import webtest
+from sqlalchemy import exc
+from dcicutils.ff_mocks import NO_SERVER_FIXTURES
+
+from dcicutils.qa_utils import notice_pytest_fixtures, MockFileSystem
+from pyramid.request import apply_request_extensions
+from pyramid.testing import DummyRequest
+from pyramid.threadlocal import get_current_registry, manager as threadlocal_manager
+from snovault import DBSESSION, ROOT, UPGRADER
+from snovault.elasticsearch import ELASTIC_SEARCH, create_mapping
+from snovault.util import generate_indexer_namespace_for_testing
+from snovault.loadxl import load_all
+from snovault.tests.serverfixtures import (
+    wsgi_server_host_port, conn, DBSession, _DBSession, engine, engine_url, zsa_savepoints, check_constraints
+)
```

Comment:
This import was not in `cgap-portal` and I think should not be needed. These fixtures should be available anyway, and what makes that happen is making sure there is a proper `-p` option automatically added in the `pytest.ini`. That seems to be missing, but you can compare to `cgap-portal`'s `pytest.ini`. I'll make a note in the right place in this PR, too, though.